### PR TITLE
Fixed invalid path in default schedule event

### LINKED ISSUE #873: Missing leading slash in default schedule path
This is a about [this](http://edgex-core-data:48080api/v1/event/scrub) line in the scheduler docker config file.
The support-scheduler generates schedules from its config.
Currently:
```
 Path = 'api/v1/event/scrub'
```
This results in the following addressable path:
```
http://edgex-core-data:48080api/v1/event/scrub
```
Where clearly the slash after the port is missing.

Edit: Consul is disabled

### PR DESCRIPTION
**Pull request by jduranf**
Fix #873

Signed-off-by: Joan Duran <jduran@circutor.com>

### CODE REVIEW

File: cmd/support-scheduler/res/docker/configuration.toml [hunk 1]
```diff
@@ -59,9 +59,3 @@ File = '/edgex/logs/edgex-support-scheduler.log'
     Path = 'api/v1/event/removeold/age/604800000'
```
Comment (nfode) [inline cmd/support-scheduler/res/docker/configuration.toml]: Just realized its the same for this line
```suggestion
    Path = '/api/v1/event/removeold/age/604800000'
```

Comment (jduranf) [reply cmd/support-scheduler/res/docker/configuration.toml]: You are right, fixed too

**Comment by tsconn23** [comment]
recheck

**Comment by tsconn23** [comment]
@jduranf Do you have time to duplicate this PR targeting the `delhi` branch? Would like to have this in the 0.7.1 release.

File: cmd/support-scheduler/res/docker/configuration.toml (hunk 1 above)
Comment (xmlviking) [reply cmd/support-scheduler/res/docker/configuration.toml]: Is this fixed now?
If so having this in the Delhi would be fantastic

**Comment by jduranf** [comment]
@tsconn23 created #886 for Delhi

**Pull request merged**